appGeo/src/app/home/home-routing.module.ts

```diff
@@ -1,18 +1,18 @@
 import { NgModule } from '@angular/core'
 import { Routes } from '@angular/router'
 import { NativeScriptRouterModule } from '@nativescript/angular'
-import { Chat } from '../chat/chat.component'
+import { ChatComponent } from '../chat/chat.component'
 //The Chat is added here so that the template can be represented
 import { CampusMap } from '../map/campus-map.component'
 import {Bubble} from '../map/map.component'
 
 import { HomeComponent } from './home.component'
 
 const routes: Routes = [
+  {path: "Chat", component: ChatComponent},
   { path: '',
-    component: HomeComponent,
-    children: [
-      {path: "Chat", component: Chat}] },
+    component: HomeComponent
+  },
```

Comment:
Please note: this is where the issue with the button being overlayed and a new page _not_ being created when clicking button for Chat was occurring.